lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -1,9 +1,34 @@
 import { RenovateConfig } from '../../../../config';
+import { getPreset } from '../../../../config/presets';
+import { PRESET_DEP_NOT_FOUND } from '../../../../config/presets/util';
 import { logger } from '../../../../logger';
 import { clone } from '../../../../util/clone';
 
-export function getOnboardingConfig(config: RenovateConfig): string {
-  const onboardingConfig = clone(config.onboardingConfig);
+export async function getOnboardingConfig(
+  config: RenovateConfig
+): Promise<string> {
+  let onboardingConfig = clone(config.onboardingConfig);
+
+  let organizationConfigRepoExists = false;
+  const organizationConfigPresetName = `local>${config.owner}/renovate-config`;
+
+  try {
+    if (config.owner) {
+      await getPreset(organizationConfigPresetName, config);
+      organizationConfigRepoExists = true;
+    }
+  } catch (err) {
+    if (err.message !== PRESET_DEP_NOT_FOUND) {
+      throw err;
+    }
+    // Organization preset did not exist
+  }
+  if (organizationConfigRepoExists) {
+    onboardingConfig = {
+      extends: [organizationConfigPresetName],
```

Comment:
```suggestion
      $schema: 'https://docs.renovatebot.com/renovate-schema.json',
      extends: [organizationConfigPresetName],
```

Reply:
This has caused CI to fail: `Error: lib/workers/repository/onboarding/branch/config.ts(28,7): error TS2322: Type '{ $schema: string; extends: string[]; }' is not assignable to type 'RenovateSharedConfig'.`
https://github.com/renovatebot/renovate/pull/7403/checks?check_run_id=1210266145

I don't understand the various Renovate*Config types - could you help with which type needs the `$schema` key added to it?

Reply:
https://github.com/renovatebot/renovate/blob/243728f2637f92076287adda6e964025ed3e6195/lib/config/common.ts#L20

add a `$schema?: string;` line there.